pkg/permission/manager/ocisci/ocisci.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021 CERN
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// In applying this license, CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+package ocisci
+
+import (
+	provider "github.com/cs3org/go-cs3apis/cs3/storage/provider/v1beta1"
+	"github.com/cs3org/reva/pkg/permission"
+	"github.com/cs3org/reva/pkg/permission/manager/registry"
+)
+
+func init() {
+	registry.Register("ocisci", New)
+}
+
+// New returns a new permission manager specific for the CI
+func New(c map[string]interface{}) (permission.Manager, error) {
+	return manager{}, nil
+}
+
+type manager struct {
+}
+
+func (m manager) CheckPermission(permission string, subject string, ref *provider.Reference) bool {
+	// We can currently return false all the time.
+	// Once we beginn testing roles we need to somehow check the roles of the users here
+	return false
```

Comment:
I'd rather return true by default. You can get rid of the `checkRes.Status.Code == v1beta11.Code_CODE_OK` check in the `ListStorageSpaces` in decomposedfs then as well

Reply:
Not really. The response contains the status `ok` or `permission denied` depending of if the user has the permission.
In hindsight I think this is bad API design and there should be a dedicated `result` field since the status is more concerning the transport.
But I'd change that afterwards to not delay this PR further.